app/modules/import-export/usage.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Elementor\App\Modules\ImportExport;
+
+use Elementor\App\Modules\ImportExport\Processes\Revert;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Usage {
+
+	/**
+	 * Register hooks.
+	 *
+	 * @return void
+	 */
+	public function register() {
+		add_filter( 'elementor/tracker/send_tracking_data_params', function ( array $params ) {
```

Comment:
I would just split it into another function, to avoid the anonymous function.
```suggestion
		add_filter( 'elementor/tracker/send_tracking_data_params', [ $this, 'add_tracking_data' ] );
```

Reply:
That will force me to publicize my function, I prefer not to.

Reply:
Ok, no problem.